internal/cmd/kafka/command_mirror_list.go

```diff
@@ -28,13 +29,13 @@ func (c *mirrorCommand) newListCommand() *cobra.Command {
 			},
 			examples.Example{
 				Text: `List all active mirror topics under "my-link":`,
-				Code: "confluent kafka mirror list --link my-link --mirror-status active",
+				Code: "confluent kafka mirror list --link my-link --mirror-status ACTIVE",
```

Comment:
Can we just upper-case whatever the user passes in? I'd like to keep all of our examples lower-kebab-case.